src/domain/event/hooks/utils.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  EventFieldsFragment,
+  PublicationStatus,
+} from '../../../generated/graphql';
+import { OptionType } from '../../../types';
+import parseIdFromAtId from '../../../utils/parseIdFromAtId';
+import { AUDIENCE_ORDER, EVENT_EDIT_ACTIONS } from '../constants';
+
+export const getEventUpdateAction = (
+  event: EventFieldsFragment,
+  publicationStatus: PublicationStatus
+): EVENT_EDIT_ACTIONS =>
+  event.publicationStatus === PublicationStatus.Draft
+    ? publicationStatus === PublicationStatus.Draft
+      ? EVENT_EDIT_ACTIONS.UPDATE_DRAFT
+      : EVENT_EDIT_ACTIONS.PUBLISH
+    : EVENT_EDIT_ACTIONS.UPDATE_PUBLIC;
```

Comment:
To me this nested ternary feels little bit hard to understand

Reply:
Changed to 
```
export const getEventUpdateAction = (
  event: EventFieldsFragment,
  publicationStatus: PublicationStatus
): EVENT_EDIT_ACTIONS => {
  if (event.publicationStatus === PublicationStatus.Draft) {
    return publicationStatus === PublicationStatus.Draft
      ? EVENT_EDIT_ACTIONS.UPDATE_DRAFT
      : EVENT_EDIT_ACTIONS.PUBLISH;
  }
  return EVENT_EDIT_ACTIONS.UPDATE_PUBLIC;
};
```